File: Service/TicketService.cs
using System.Collections.Generic;
using Domain;
using projectC.Repository;

namespace projectC.Service
{
    public class TicketService
    {
        private readonly TicketRepository ticketRepository;

        public TicketService(TicketRepository ticketRepository)
        {
            this.ticketRepository = ticketRepository;
        }

        public List<Ticket> GetTickets()
        {
            return ticketRepository.GetAll();
        }

        public void AddTicket(Ticket ticket)
        {
            ticketRepository.Add(ticket);
        }

        public void DeleteTicket(int ticketId)
        {
            ticketRepository.Remove(ticketId);
        }

        public void UpdateTicket(int ticketId, Ticket ticket)
        {
            ticketRepository.Update(ticketId, ticket);
        }
    }
}

File: project-java-BogdanBala112/HomeAssignment1/src/main/java/service/TicketService.java
package service;

import Entities.Ticket;
import Repository.TicketRepository;

import java.util.List;

public class TicketService {
    public TicketRepository ticketRepository;

    public TicketService(TicketRepository ticketRepository) {
        this.ticketRepository = ticketRepository;
    }

    public List<Ticket> getTickets(){
        return ticketRepository.getAll();
    }

    public void addTicket(Ticket ticket){
        ticketRepository.add(ticket);
    }

    public void deleteTicket(Integer id){
        ticketRepository.remove(id);
    }

    public void updateTicket(int id, Ticket ticket){
        ticketRepository.update(id, ticket);
    }
}
